### PR TITLE
Adds statuspage.io documentation to the manual

### PR DESCRIPTION
**Pull request by LeePorte**
## What

As we have just started using statuspage.io, this is some initial
documentation to aid in getting started and some knowledge transfer.
There are notes for future enhancements to the page.

## How to review

Read the docs :) Feel free to login to statuspage and verify

## Who can review?

Not me

### CODE REVIEW

File: docs/team/statuspage.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
+The team uses [StatusPage.io](https://www.statuspage.io/) to notify our users of platform issues.
+
+## Accounts
+
+We have a team user. Its credentials are stored in the credentials repository.
+
+We create individual users for those that will use it as a communication tool.
```
Comment (timmow) [inline docs/team/statuspage.md]: What does "use it as a communication tool" mean? Maybe we should say something about users posting status updates?

File: docs/team/statuspage.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+Incidents have one of four distinct statuses. When creating or updating an incident you will specify which of the states the incident is currently in.
+
+- Investigating - you are seeing the symptoms of an issue but are unaware what the root cause.
+- Identified - you have found the root cause of the incident and are working on a fix.
+- Monitoring - you believe you have successfully fixed the issue and are waiting for the symptoms to subside.
+- Resolved - the root cause of the issue has been eliminated and your systems are back to 100% performance.
+
+#### Message
+
+When creating or updating an incident, you will also give a brief message explaining the current situation. This field accepts HTML.
```
Comment (timmow) [inline docs/team/statuspage.md]: This looks like its pulled directly from statuspage.ios docs - Its probably better to link to it if this is the case

File: docs/team/statuspage.md (hunk 1 above)
Comment (LeePorte) [inline docs/team/statuspage.md]: This is one to discuss with the Product management team and refine based on the outcome of the discussion.

File: docs/team/statuspage.md (hunk 2 above)
Comment (LeePorte) [inline docs/team/statuspage.md]: It pretty much is, I'll update.

File: docs/team/statuspage.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
+The team uses [StatusPage.io](https://www.statuspage.io/) to notify our users of platform issues and maintenance windows. As this provides simple, clear and concise information related to the health and status of the platform. It also enables webhook access for tenants to link to Slack / HipChat / Link. 
+
+The Announce mailing list is to be used to announce new features and any resulting changes to tenants.
+
+## Accounts
+
+We have a team user for technical support of ststuspage.io and for any issues that arise whilst on support. Its credentials are stored in the credentials repository.
```
Comment (timmow) [inline docs/team/statuspage.md]: typo - ststuspage should be statuspage

**Pull request merged**